recipe/meta.yaml

```diff
@@ -23,30 +22,34 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python
+    - cython
     - setuptools
     - pip
     - wheel
   run:
-    - python >=3.6
-    # for fontTools.sfnt and fontTools.woff2: to compress/uncompress
-    # WOFF 1.0 and WOFF 2.0 webfonts.
+    - python
     - brotli >=1.0.1
-    # 5/17/2021: the unicodedata2 extension module doesn't work on PyPy.
-    # Python 3.9 already has Unicode 13.0, so the backport is not needed.
-    - unicodedata2 >=13.0.0  # [py<39 and (python_impl != "pypy")]
-    - munkres
+    - unicodedata2 >=15.1.0 # [py<313]
+  run_constrained:
+    - lxml >=4.0
+    - lz4 >=1.7.4.2
+    - skia-pathops >=0.5.0
+    - uharfbuzz >=0.23.0
 
 test:
   imports:
     - fontTools
     - fontTools.ttLib
   commands:
+    - pip check
     - fonttools subset --help
```

Comment:
Could you please test these commands locally:
```
 - python -c "from fontTools.cu2qu.cu2qu import COMPILED; assert COMPILED"
 - python -c "from fontTools.cu2qu.cu2qu import COMPILED; assert not COMPILED"
```

See https://github.com/fonttools/fonttools/blob/705acc994f9f8785a582e981a4913394dc7699c0/tox.ini#L26-L27

Reply:
Thank you very much. I added both commands, after setting the environment variable. First command worked, and second command failed once it compiled correctly.